src/js/jsx/sections/style/ShadowList.jsx

```diff
@@ -219,7 +219,7 @@ define(function (require, exports) {
                     blendMode,
                     this.props.type);
             // Currently does not differentiate between inner and drop shadows
-            headlights.logEvent("edit", "shadow-blendmode-input", blendMode);
+            headlights.logEventDebounced("edit", "shadow-blendmode-input", blendMode);
```

Comment:
I don't think this is a good pattern, btw. If you call `logEventDebounced` in multiple places with different arguments at nearly the same time, all but the last call will be dropped... It might be OK in this case, but...